pkg/metastore/glue/converter.go

```diff
@@ -0,0 +1,269 @@
+		Name:        aws.StringValue(db.Name),
+		Description: aws.StringValue(db.Description),
+		LocationURI: aws.StringValue(db.LocationUri),
+		Parameters:  aws.StringValueMap(db.Parameters),
+	}
+}
+
+func TableLocalToGlue(table *metastore.Table) *glue.TableInput {
+	sd := SDLocalToGlue(table.Sd)
+	targetTable, _ := table.AWSTargetTable.(*glue.TableIdentifier)
+	ht := &glue.TableInput{
+		Description:       table.AWSDescription,
+		LastAccessTime:    localToAWSTime(table.LastAccessTime), // TODO(Guys): check if this OK
+		LastAnalyzedTime:  table.AWSLastAnalyzedTime,
+		Name:              aws.String(table.TableName),
+		Owner:             aws.String(table.Owner),
+		Parameters:        aws.StringMap(table.Parameters),
+		PartitionKeys:     columnsLocalToGlue(table.PartitionKeys),
+		Retention:         aws.Int64(int64(table.Retention)), // TODO(Guys): check if this OK
+		StorageDescriptor: sd,
+		TableType:         aws.String(table.TableType),
+		TargetTable:       targetTable,
+		ViewExpandedText:  aws.String(table.ViewExpandedText),
+		ViewOriginalText:  aws.String(table.ViewOriginalText),
+	}
+	return ht
+}
+
+func PartitionLocalToGlue(partition *metastore.Partition) *glue.PartitionInput {
+	sd := SDLocalToGlue(partition.Sd)
+	ht := &glue.PartitionInput{
+		//IsRegisteredWithLakeFormation: partition.AWSIsRegisteredWithLakeFormation,
+		LastAccessTime:    localToAWSTime(int64(partition.LastAccessTime)), // TODO(Guys): check if this OK
+		LastAnalyzedTime:  partition.AWSLastAnalyzedTime,
+		Parameters:        aws.StringMap(partition.Parameters),
+		StorageDescriptor: sd,
+		Values:            aws.StringSlice(partition.Values),
+	}
+	return ht
+}
+
+func PartitionsLocalToGlue(partitions []*metastore.Partition) []*glue.PartitionInput {
+	gluePartitions := make([]*glue.PartitionInput, len(partitions))
+	for i, partition := range partitions {
+		gluePartitions[i] = PartitionLocalToGlue(partition)
+	}
+	return gluePartitions
+}
+
+func SDLocalToGlue(sd *metastore.StorageDescriptor) *glue.StorageDescriptor {
+	if sd == nil {
+		return nil
+	}
+	schemaRef, _ := sd.AWSSchemaReference.(*glue.SchemaReference)
+	return &glue.StorageDescriptor{
+		BucketColumns:          aws.StringSlice(sd.BucketCols),
+		Columns:                columnsLocalToGlue(sd.Cols),
+		Compressed:             aws.Bool(sd.Compressed),
+		InputFormat:            aws.String(sd.InputFormat),
+		Location:               aws.String(sd.Location),
+		NumberOfBuckets:        aws.Int64(int64(sd.NumBuckets)),
+		OutputFormat:           aws.String(sd.OutputFormat),
+		Parameters:             aws.StringMap(sd.Parameters),
+		SchemaReference:        schemaRef,
+		SerdeInfo:              serDeLocalToGlue(sd.SerdeInfo),
+		SkewedInfo:             skewedLocalToGlue(sd.SkewedInfo),
+		SortColumns:            sortColumnsLocalToGlue(sd.SortCols),
+		StoredAsSubDirectories: sd.StoredAsSubDirectories,
+	}
+}
+
+func SDGlueToLocal(sd *glue.StorageDescriptor) *metastore.StorageDescriptor {
```

Comment:
do we need to address `SchemaReference` also in this func as we do in SDLocalToGlue?